### DISCUSSION TITLE
Equivalent to Html::attributes?

### DISCUSSION THREAD

**Discussion by schonhose** [spatie/laravel-html#230] [Q&A]
Is there an equivalent function available for `Html::attributes`, as defined in `laravelcollective/Html` [here](https://github.com/LaravelCollective/html/blob/6.x/src/HtmlBuilder.php#L426)?

**Comment by bskl** [comment]
You can use this method.

[https://spatie.be/docs/laravel-html/v3/general-usage/element-methods#content-attributes](https://spatie.be/docs/laravel-html/v3/general-usage/element-methods#content-attributes)
